aurget

```diff
@@ -717,7 +728,9 @@ declare -A target_versions # name->version
 declare -A target_deps     # pkgbase->is-dep?
 
 initialize "$@"
-
+if clean; then
+clean_package_directory
+fi
 if searching; then
```

Comment:
Do you intend for the behavior of `--clean` to be that you clean and can proceed with other operations, or do you want `--clean` to be something that has to be used by itself?

If it's the former, that shouldn't be represented by `opmode=clean` it should a boolean variable like `clean=true`. Then you can just do `clean && clean_...` here.

If it's the latter, this should be `if; clean; then; elif searching; then; else` -- or maybe a `case`?

Reply:
Which would be more preferable to you?

Reply:
I don't have a a strong opinion about it. I'd probably do the former if it were up to me, but that's a bigger change from the code you have here now.